features/google/src/components/awakeningLocationMinistory/awakeningLocationMinistory.jsx

```diff
@@ -0,0 +1,135 @@
+import "./awakeningLocationMinistory.css";
+import React, { useState } from "react";
+import SourceInfoButton from "../sourceInfoButton/sourceInfoButton.jsx";
+import i18n from "!silly-i18n";
+
+export const AwakeningLocationSummary = ({ dateData }) => {
+    const defaultDate = Object.entries(dateData).find(
+        (entry) => !!entry[1].firstActivity
+    )[0];
+    const [selectedDate, setSelectedDate] = useState(defaultDate);
+    const data = dateData[selectedDate];
+    const dateSelection = false;
+    const handleChangeDate = ({ target }) => {
+        const formattedDate = target.value
+            .split("-")
+            .reverse()
+            .map((e) => parseInt(e))
+            .join("-");
+        setSelectedDate(formattedDate);
+        console.log(formattedDate);
+    };
+    //could be used to make the text more human later on
+    // const week = [
+    //     "Sunday",
+    //     "Monday",
+    //     "Tuesday",
+    //     "Wednesday",
+    //     "Thursday",
+    //     "Friday",
+    //     "Saturday",
+    // ];
+    return (
+        <div className="awakening-location-ministory-summary ">
+            {dateSelection ? (
+                <div className="date-browser">
+                    <label>
+                        Enter a date:
+                        <input
+                            className="poly-theme-light"
+                            type="date"
+                            name="selectedDate"
+                            max={Object.keys(dateData).pop()}
+                            min={Object.keys(dateData)[0]}
+                            required
+                            pattern="\d{2}-\d{2}-\d{4}"
```

Comment:
Aren't slashes more common? Even if they are, shouldn't you say where month should go? And wouldn't it be a bit more flexible `\d{1,2}` in both cases?

Reply:
sure thing :)